core/trino-server-rpm/src/main/rpm/postremove

```diff
@@ -1,20 +1,19 @@
 # Post erase script
 
 # if this is the last version of the rpm being removed (i.e. not on upgrade)
-if [ "$1" -eq 0 ]; then
-    # Delete /etc/trino/env.sh manually during uninstall.
-    # rpm -e wont remove it, because this file is created during preinstall
-    rm -f /etc/trino/env.sh
-    # Delete the data directory manually during uninstall.
-    # rpm -e wont remove it, because this directory may later contain files not
-    # deployed by the rpm
-    rm -rf /var/lib/trino
-    # The RPM instead of linking to /usr/lib/trino/ as a directory attribute
-    # now links to all the individual jar files in /usr/lib/trino/lib/ and
-    # /usr/lib/trino/plugin/. So remove the parent dir manually during uninstall
-    rm -rf /usr/lib/trino
-    # Remove /etc/trino directory if no other files present
-    if [ -z "$(ls -A /etc/trino)" ]; then
-        rm -rf /etc/trino
-    fi
+if [ "$1" -ne 0 ]; then
+    exit 0
+fi
+
+# Delete the data directory manually during uninstall.
```

Comment:
Should this be part of fmt commit?

Reply:
I removed deleting of `env.sh` here but it's hard to spot since the fmt commit lacked some refactor. Fixed!